server/server.go

```diff
@@ -147,7 +151,7 @@ func (s *Server) LoadBlob(t pack.BlobType, id backend.ID) ([]byte, error) {
 
 // LoadJSONEncrypted decrypts the data and afterwards calls json.Unmarshal on
 // the item.
-func (s *Server) LoadJSONEncrypted(t backend.Type, id backend.ID, item interface{}) error {
+func (s *Server) LoadJSONUnpacked(t backend.Type, id backend.ID, item interface{}) error {
```

Comment:
I kinda liked the old name better.. it's not clear to me anymore that the data is encrypted.. I would change it back (or actually I would prefer LoadEncryptedJSON, which is even more clear, imho).